src/com/owncloud/android/utils/DisplayUtils.java

```diff
@@ -207,8 +207,17 @@ else if ((System.currentTimeMillis() - time) < 60 * 1000) {
                 dateString = DateUtils.getRelativeDateTimeString(c, time, minResolution, transitionResolution, flags);
             }
         }
-        
-        return dateString.toString().split(",")[0];
+
+        String[] parts = dateString.toString().split(",");
+        if (parts.length == 2) {
+            if(parts[0].contains(":") && !parts[1].contains(":")) {
+                return parts[1];
+            } else if(parts[1].contains(":") && !parts[0].contains(":")) {
+                return parts[0];
+            }

```

Comment:
Good point, swapped it in commit https://github.com/owncloud/android/commit/e8eda963ae6aaf3ac73500c4b75a3c2d533b6176